lib/trento/clusters.ex

```diff
@@ -141,7 +142,50 @@ defmodule Trento.Clusters do
   defp enrich_cluster_model_query(query) do
     query
     |> join(:left, [c], e in ClusterEnrichmentData, on: c.id == e.cluster_id)
-    |> select_merge([c, e], %{cib_last_written: e.cib_last_written})
+    |> select_merge([c, e], %{cib_last_written: e.cib_last_written, enriching_details: e.details})
```

Comment:
If you want to remove the `virtual` field, you could skip this join, and simply query the details in `enrich_cluster_details`. We actually do the same in `enrich_cluster_model`. 
This way we don't have the risk to slip away this field anywhere.
In fact, we could reuse the function in other parts (I think we need to do it in `ClusterRegistered` projection, where we use `enrich_cluster_model`

It adds an extra query, but it removes the virtual field and its inconveniences.